specs/Deployment/DeploymentAPI.md

```diff
@@ -179,12 +186,43 @@ they shut down, to refer to the updated framework package.
     }
 ```
 
+## Repair
+
+This API performs a status check, and if the main and singleton packages are already installed at
+the required version, it will attempt to repair those packages. All information about the version,
+channel, and architecture needed are derived from the current WinAppSDK framework package.
+
+Since both the main _and_ the singleton packages may need to be repaired, it is possible that one
+package is repaired successfully but not the other. If so, the returned WindowsAppRuntimeStatus will
+contain the error, if one occurs during package install.
+
+```C#
+    if (DeploymentManager.GetStatus().Status != DeploymentStatus.Ok)
+    {
+        // Repair does a status check, and if the status is OK it will attempt to get
+        // the WindowsAppRuntime into a good state by deploying packages. Unlike a simple
+        // status check, Repair can sometimes take several seconds to deploy the packages.
+        // These should be run on a separate thread so as not to hang your app while the
+        // packages deploy.
+        var repairTask = Task.Run(() => DeploymentManager.Repair());
+        repairTask.Wait();
```

Comment:
You're kicking the Repair call to a separate task and then immediately .Wait()'ing for it. That makes it kinda pointlerss, no? Is this the same as `var repairResult = DeploymentManager.Repair().get();` but faster / less overhead?

If the intent is to do other work while Repair is running this should be

```c#
var repairTask = ...
...do other work while the repair is running...
repairTask.Wait();
```

Reply:
I think separate task was used so that a hang in .get() doesn't cause a hang in the main thread of the app. My understanding is .get() is supposed to wait indefinitely for deployment to complete the request. Hence, this extra care for the app.

Reply:
As an example it makes more sense with the "...do other work..." comment between the task start + wait

Lacking that it's kinda pointless to do the task. This thread is going to kick off the task and then block for .Wait to return. Net result this thread will "hang"/block. No?

Reply:
Added "...do other work..." comment between the task start + wait